assignments/objects.js

```diff
@@ -11,54 +11,93 @@
-// Gannie's name
-
-// Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
-// Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
-
-// Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
-
-// === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
-
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
-
-// 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
-// 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
-// 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
-// 4. Give each of the objects the ability to speak their names using the this keyword.
-
-let parent = {}
-
-// Log the parent object's name
-
-// Log the child's age
-
-// Log the name and age of the grandchild
-
-// Have the parent speak
-
-// Have the child speak
-
-// Have the grandchild speak
+    "id": 0,
+    "name": "Example",
+    "email": "examples@you.edu",
+    "gender": "F"
+  }
+  
+  // Write your intern objects here:
+  let intern1 = {
+      "id": 1,
+      "name": "Mitzi",
+      "email": "mmelloy0@psu.edu",
+      "gender": "F"
+  }
+  let kennan = {
+      "id": 2,
+      "name": "Kennan",
+      "email": "kdiben1@tinypic.com",
+      "gender": "M"
+  }
+  let intern3 = {
+      "id": 3,
+      "name": "Keven",
+      "email": "kmummery2@wokimedia.org",
+      "gender": "M"
+  }
+  let intern4 = {
+      "id": 4,
+      "name": "Gannie",
+      "email": "gmartinson3@illinois.edu",
+      "gender": "M"
+  }
+  let intern5 = {
+      "id": 5,
+      "name": "Antonietta",
+      "email": "adaine5@samsung.com",
+      "gender": "F"
+  }
+  
+  // ==== Challenge 2: Reading Object Data ==== 
+  // Once your objects are created, log out the following requests from HR into the console:
+  
+  // Mitzi's name
+  console.log(intern1["name"])
+  // Kennan's ID
+  console.log(kennan["id"])
+  // Keven's email
+  console.log(intern3["email"])
+  // Gannie's name
+  console.log(intern4["name"])
+  // Antonietta's Gender
+  console.log(intern5["gender"])
+  // ==== Challenge 3: Object Methods ==== 
+  // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
+  // console.log(kennan.speak());
+
+  kennan.speak = function(){
+ console.log("Hello, my name is Kennan!");
+    };
+    console.log(kennan.speak());
+  
+  // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
+  //console.log(antonietta.multiplyNums(3,4));
+  intern4.multiplyNums = function(a,b){
+  console.log(a*b);
```

Comment:
this is good.  its generally best practices to have some spacing  i.e. (a * b) just helps witih readability.